content/posts/irods-development-update-january-2023.md

```diff
@@ -0,0 +1,240 @@
+Title: iRODS Development Update: January 2023
+Date: 2023-01-23 12:00
```

Comment:
Will bump once we're happy with the content.

Reply:
yep - pick 1600 and we'll do it.

Reply:
Done.